R/importNCRN.R

```diff
@@ -11,25 +11,23 @@
 
 
 importNCRN<-function(Dir){
-  OldDir<-getwd()
-  setwd(Dir)  
-
-  InPlots<-read.csv("Plots.csv",as.is=T, header=T)
+  
+  InPlots<-read.csv(paste(Dir,"Plots.csv",sep="/"),as.is=T, header=T)
   InPlots$Event_Earliest<-as.Date(as.character(InPlots$Event_Earliest), format="%Y%m%d")
   InPlots$Event_Latest<-as.Date(as.character(InPlots$Event_Latest),format="%Y%m%d")
   
-  InEvents<-read.csv("Events.csv",as.is=T, header=T)
+  InEvents<-read.csv(paste(Dir,"Events.csv",sep="/"),as.is=T, header=T)
   InEvents$Event_Date<-as.Date(as.character(InEvents$Event_Date_Txt), format="%Y%m%d")
   
   
-  InTrees<-read.csv("Trees.csv",as.is=T, header=T)
-  InSaps<-read.csv("Saplings.csv",as.is=T, header=T)
-  InSeeds<-read.csv("Seedlings.csv",as.is=T, header=T)
-  InShrubs<-read.csv("Shrubs.csv",as.is=T, header=T)
-  InShSeeds<-read.csv("Shrub_Seedlings.csv",as.is=T, header=T)
-  InVines<-read.csv("Vines.csv",as.is=T, header=T)
-  InHerbs<-read.csv("Herbs.csv",as.is=T, header=T)
-  InCommons<-read.csv("CommonNames.csv", as.is=T, header=T)
+  InTrees<-read.csv(paste(Dir,"Trees.csv",sep="/"),as.is=T, header=T)
+  InSaps<-read.csv(paste(Dir,"Saplings.csv",sep="/"),as.is=T, header=T)
+  InSeeds<-read.csv(paste(Dir,"Seedlings.csv",sep="/"),as.is=T, header=T)
+  InShrubs<-read.csv(paste(Dir,"Shrubs.csv",sep="/"),as.is=T, header=T)
+  InShSeeds<-read.csv(paste(Dir,"Shrub_Seedlings.csv",sep="/"),as.is=T, header=T)
+  InVines<-read.csv(paste(Dir,"Vines.csv",sep="/"),as.is=T, header=T)
+  InHerbs<-read.csv(paste(Dir,"Herbs.csv",sep="/"),as.is=T, header=T)
+  InCommons<-read.csv(paste(Dir,"CommonNames.csv",sep="/"), as.is=T, header=T)
   InCommons$Common[InCommons$NCRN_Common!=""]<-InCommons$NCRN_Common[InCommons$NCRN_Common!=""]
   InCommons$TSN<-as.character(InCommons$TSN)
   setwd(OldDir)
```

Comment:
This line should be deleted

Reply:
Snap! Sorry for not catching that. For some reason I thought that line was at the very end of the function, so missed that it was still there. I went through and deleted them, and pushed the files.